src/components/Menu/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { IMenu } from "./types";
+import menuOptions from "./const";
+//Translate - i18next
+import { useTranslation } from "react-i18next";
+//styles components
+import { StyledDiv, StyledMenu, StyledOptions, StyledUser } from "./styles";
+//images
+import UserPic from "../../assets/img/user-avatar.svg";
+import Close from "../../assets/img/close-x.svg";
+import { useState } from "react";
+
+const Menu: React.FC<IMenu> = ({ isOpen, userName, userType }: IMenu) => {
+    const { t } = useTranslation();
+
+    const [IsOpen, setIsOpen] = useState(isOpen);
```

Comment:
```suggestion
    const [isOpen, setIsOpen] = useState(isOpen);
```
Holi! Acá la convensión es usar camelCase para los useState. sería isOpen con minúscula, no sé si lo pusiste porque no le gusta que sea el mismo nombre que la prop? queda medio mal así con mayuss jejej. quizás pensar en otro nombre para la prop.. así no se chocan

Reply:
Ohh juraba que eso estaba en minúscula :O Ahí lo cambio, gracias!